kfs-purap/src/main/java/edu/arizona/kfs/module/purap/document/web/struts/PaymentRequestAction.java

```diff
@@ -0,0 +1,96 @@
+package edu.arizona.kfs.module.purap.document.web.struts;
+
+import java.text.MessageFormat;
+import java.util.ArrayList;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.log4j.Logger;
+import org.apache.struts.action.ActionForm;
+import org.apache.struts.action.ActionForward;
+import org.apache.struts.action.ActionMapping;
+import org.kuali.kfs.module.purap.document.PaymentRequestDocument;
+import org.kuali.kfs.module.purap.document.PurchaseOrderDocument;
+import org.kuali.kfs.module.purap.document.web.struts.PaymentRequestForm;
+import org.kuali.kfs.sys.context.SpringContext;
+import org.kuali.rice.core.api.config.property.ConfigurationService;
+import org.kuali.rice.kns.question.ConfirmationQuestion;
+
+import edu.arizona.kfs.fp.service.DisbursementVoucherInvoiceService;
+import edu.arizona.kfs.module.purap.PurapConstants;
+import edu.arizona.kfs.module.purap.PurapKeyConstants;
+import edu.arizona.kfs.sys.KFSConstants;

```

Comment:
This import is not needed. None of the constants in edu.arizona.kfs.sys.KFSConstants are used in this class.